### PR TITLE
added doc part for multiupload

### PR DESCRIPTION
**Pull request by silasjoisten**
Just a docs update

### CODE REVIEW

File: docs/reference/extra.rst [hunk 1]
```diff
@@ -206,3 +206,18 @@ Here is an example:
     ));
 
 And that's it, enjoy browsing and uploading your medias using ``SonataMediaBundle``.
+
+Sonata Multiupload Bundle
+======================================
```
Comment (kunicmarko20) [inline docs/reference/extra.rst]: please remove extra `=`

File: docs/reference/extra.rst [hunk 2]
```diff
@@ -206,3 +206,18 @@ Here is an example:
...
+
+This bundle allows you to upload a stack of Media's. It provides JS
```
Comment (kunicmarko20) [inline docs/reference/extra.rst]: `a stack of` => `multiple`?
`JS` => `JavaScript`?

File: docs/reference/extra.rst [hunk 3]
```diff
@@ -206,3 +206,18 @@ Here is an example:
...
+side mimetype and file extension validation.
+
+Take a look:
```
Comment (kunicmarko20) [inline docs/reference/extra.rst]: don't think this is needed

File: docs/reference/extra.rst [hunk 4]
```diff
@@ -206,3 +206,18 @@ Here is an example:
...
+
+.. figure:: ../images/multiupload-bundle.gif
+   :align: center
+   :alt: Multiupload Bundle
+
+If you like it check it out
```
Comment (kunicmarko20) [inline docs/reference/extra.rst]: `If you like it check it out` => `Read the full documentation here<add link here>`?

**Comment by OskarStark** [comment]
Thank you

**Pull request merged**